Insights/IP/IP_SessionCounts.yaml

```diff
@@ -93,6 +93,12 @@ BaseQuery: |
   //| where not(DeviceAction has_any ('built','deny', 'denied', 'rst', 'blocked', 'teardown'))
   | extend Direction = iff(CommunicationDirection !in~ ('outbound','0') or CommunicationDirection !in~ ('inbound','1'), 'NotAvailable', CommunicationDirection)
   | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), InboundCount = countif(Direction in~ ('Inbound','1')), OutboundCount = countif(Direction in~ ('Outbound', '0')), UnknownDirection = countif(Direction =~ 'NotAvailable') by IPAddress = DestinationIP, Type = strcat(Type,':', DeviceVendor,'-', DeviceProduct), RemoteIPAddress = SourceIP, Direction, SentBytes = tolong(SentBytes), ReceivedBytes = tolong(ReceivedBytes)
+  ),
+  (
+  imNetworkSession(starttime=start_time, endtime=end_time, dstipaddr=Ip_Address) // , dvcaction='Allow')
+  | where SrcIpAddr != DstIpAddr
```

Comment:
How do you think unknown direction would work?
How do we handle duplicates?

Reply:
Direction: I think unknown direction needs to be used on conjunction with the source data type.  In other words, not all data types provide direction or it is implied based on the data type.  I think we make that clear in the result to help the user.  Thoughts?

Duplicates: Seems like duplicates is a loaded term here.  For example, duplicates of an IP between a variety of logs is expected, but I don't think we should obfuscate that.  I think we could do a makeset() or pack() or the like on the Type column to aggregate but still show that this IP showed up in x,y,z data types.  Thoughts?